packages/react-components/react-positioning/src/types.ts

```diff
@@ -173,21 +170,29 @@ export interface PositioningOptions {
    * @default true
    */
   useTransform?: boolean;
+
+  /**
+   * If false, does not position anything
+   */
+  enabled?: boolean;
 }
 
 export interface PositioningProps
```

Comment:
Can you also add some comments above `PositioningProps` and `PositioningOptions` to describe the public/private situation of them and why they are separate?

Reply:
Yep :) sounds good